components/apimgt/org.wso2.carbon.apimgt.rest.api.publisher.v1/src/main/java/org/wso2/carbon/apimgt/rest/api/publisher/v1/impl/SearchApiServiceImpl.java

```diff
@@ -50,7 +50,10 @@ public class SearchApiServiceImpl implements SearchApiService {
     public Response search(Integer limit, Integer offset, String query, String ifNoneMatch,
                            MessageContext messageContext) throws APIManagementException {
         SearchResultListDTO resultListDTO = new SearchResultListDTO();
-
+        if (!query.equals(RestApiConstants.PIZZASHACK_SEARCH_QUERY) && (query.startsWith("name:") |
```

Comment:
Let's create a git issue to track a better fix for this as we discussed

Reply:
Created the git issue